app.js

```diff
@@ -1,22 +1,35 @@
 const express = require("express")
 const app = express()
-const { 
-    getTopics,
-    getArticles
+const {
+  getTopics,
+  getArticles,
+  getArticleByID
 } = require("./controllers")
 
 app.get("/api/topics", getTopics)
 app.get("/api/articles", getArticles)
+app.get("/api/articles/:article_id", getArticleByID)
+
+app.use((req, res, next) => {
+  res.status(404).send({msg: "Path not found"})
+})
 
 app.use((err, req, res, next) => {
-    if (err.status === 404) {
-      res.status(err.status).send({ message: err.message });
-    } else next(err);
+  if (err.status === 404) {
+    res.status(err.status).send({ message: err.msg });
+  } else next(err);
+});
+
```

Comment:
Line 19 is doing the same as line 14. Line 13 should include the * for all endpoints other than the ones specified in your .gets.

Reply:
added "/*" for the endpoint